schools.html

```diff
@@ -1,46 +1,102 @@
+permalink: "/partners"
 ---
 
-<h1 class="title page-title">Partnered Schools</h1>
+<h1 class="title page-title">Schools, Sponsors, and Partners</h1>
+
+<p> We'd like to give thanks to some of the amazing organizations that work with us!</p>
+
+{% assign affiliates = site.data.partners | where_exp:"item", "item.type == 'affiliate'" | sort:"name" %}
+{% assign partners = site.data.partners | where_exp:"item", "item.type == 'partner'" | sort:"name" %}
+
+<h2 class="title">Partnered Schools</h2>
 <p>
-    Currently, we are working with the following schools:
-    <ul>
-        <li>
-            <a class="tla-link" href="https://baes-lausd-ca.schoolloop.com/">Brockton Avenue Elementary School</a>
-        </li>
-        <li>
-            <a class="tla-link" href="https://rwees-compton-ca.schoolloop.com/">Ralph Waldo Emerson Elementary School</a>
-        </li>
-        <li>
-            <a class="tla-link" href="https://www.nhhs.net/">North Hollywood High School</a>
-        </li>
-        <li>
-            <a class="tla-link" href="https://www.sepulvedams.org/">Sepulveda Middle School</a>
-        </li>
-    </ul>
+    Currently, we are teaching <a href="{{basesite.url}}/classes" class="tla-link">classes</a> and working with the following schools:
+<ul>
+    <li>
+        <a class="tla-link" href="https://baes-lausd-ca.schoolloop.com/">Brockton Avenue Elementary School</a>
+    </li>
+    <li>
+        <a class="tla-link" href="https://rwees-compton-ca.schoolloop.com/">Ralph Waldo Emerson Elementary School</a>
+    </li>
+    <li>
+        <a class="tla-link" href="https://www.nhhs.net/">North Hollywood High School</a>
+    </li>
+    <li>
+        <a class="tla-link" href="https://www.sepulvedams.org/">Sepulveda Middle School</a>
+    </li>
+    <li>
+        <a class="tla-link" href="https://www.shortavenue.org/">Short Avenue Elementary School</a>
+    </li>
+</ul>
 </p>
-<h2>Previous Partnered Schools</h2>
+
 <p>
     In the past, we have worked with:
-    <ul>
-        <li> Mark Twain Middle School </li>
-        <li> Archer School for Girls </li>
-        <li> Marlborough School </li>
-    </ul>
+<ul>
+    <li> Mark Twain Middle School </li>
+    <li> Archer School for Girls </li>
+    <li> Marlborough School </li>
+</ul>
 </p>
 
+<h2>
+    Partners
+</h2>
+
 <p>
-    We're still hashing out renewing existing partnerships and seeking new ones - if you're interested in working with us, please shoot us an email at <a class="tla-link" href="mailto:{{site.email}}">{{site.email}}</a>!
+    The following organizations have directly partnered with us to further expand our organization.
+</p>
+
+<div class="team-member-grid">
+    {% for partner in partners %}
+    <div class="grid-row center-row-vertical fill-screen-row">
+        <div class="grid-col justify-center">
+            <div class="profile-img-container">
+                <img src="{{site.baseurl}}/img/partners/{{partner.path}}" class="profile-img">
```

Comment:
One note: for accessibility reasons, all `img` tags should have descriptive alt text (the `alt` prop). In our case, we can probably put `{{partner.name}} logo`!

(secondary note - for style reasons, this codebase always closes self-closing tags, like this: `<img ... />`)